RESOURCES.md

```diff
@@ -0,0 +1,46 @@
+| [eiffelevents-sdk-go-maintainers](mailto:eiffelevents-sdk-go-maintainers@googlegroups.com)               | Magnus Bäck                                                                                                         |               
+| [eiffelevents-sdk-dotnet-maintainers](mailto:eiffelevents-sdk-dotnet-maintainers@googlegroups.com)       | ?                                                                                                         |       
+| [eiffel-gerrit-plugin-maintainers](mailto:eiffel-gerrit-plugin-maintainers@googlegroups.com)             | Santosh NC                                                                                                |             
+| [eiffel-goer-maintainers](mailto:eiffel-goer-maintainers@googlegroups.com)                               | Tobias Persson                                                                                                         |                               
+| [eiffel-graphql-api-maintainers](mailto:eiffel-graphql-api-maintainers@googlegroups.com)                 | Tobias Persson                                                                                                         |                 
+| [eiffel-intelligence-maintainers](mailto:eiffel-intelligence-maintainers@googlegroups.com)               | Daniel Ståhl                                                                                              |               
+| [eiffel-jenkins-plugin-maintainers](mailto:eiffel-jenkins-plugin-maintainers@googlegroups.com)           | ?                                                                                                         |           
+| [eiffel-playground-maintainers](mailto:eiffel-playground-maintainers@googlegroups.com)                   | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |                   
+| [eiffel-pythonlib-maintainers](mailto:eiffel-pythonlib-maintainers@googlegroups.com)                     | Tobias Persson                                                                                                         |                     
+| [eiffel-remrem-maintainers](mailto:eiffel-remrem-maintainers@googlegroups.com)                           | ?                                                                                                         |                           
+| [eiffel-community-maintainers](mailto:eiffel-community-maintainers@googlegroups.com)                     | ?                                                                                                         |                     
+| [eiffel-sepia-maintainers](mailto:eiffel-sepia-maintainers@googlegroups.com)                             | ?                                                                                                         |                             

```

Comment:
```suggestion
| [eiffel-sepia-maintainers](mailto:eiffel-sepia-maintainers@googlegroups.com)                             | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members)              |                             

```

Reply:
Updated per TC notes in [8562e0b](https://github.com/eiffel-community/community/pull/179/commits/8562e0b96c4b8becdfefebcc7a7e06b3879a0f53)